tests/integration/test_cli.py

```diff
@@ -130,11 +130,7 @@ def test_pipenv_graph_reverse(pipenv_instance_private_pypi):
                                          output,
                                          flags=re.MULTILINE | re.IGNORECASE)
                 assert openpyxl_dep is None, f'openpyxl should not appear at beginning of lines in {output}'
-
-                if sys.version_info[:2] == (3, 12):
-                    assert 'openpyxl==2.5.4 [requires: et_xmlfile]' in output
-                else:
-                    assert 'openpyxl==2.5.4 [requires: et-xmlfile]' in output
+                assert 'openpyxl==2.5.4 [requires: et_xmlfile]' in output
```

Comment:
I have no idea how this straightened itself out.   I assume something to do with importlib-metadata updating, but really confusing.